src/lib/mac-address.ts

```diff
@@ -0,0 +1,20 @@
+import * as fs from 'mz/fs';
+import * as path from 'path';
+
+export async function getAll(sysClassNet: string): Promise<string | undefined> {
```

Comment:
Can you wrap this whole function in a try catch please, it means we can wrap any errors in a `MacAddress` error so it makes things more clear when this code fails.

Reply:
Nice. Done.